rotkehlchen/chain/evm/decoding/curve/curve_cache.py

```diff
@@ -493,13 +496,14 @@ def query_curve_data(
     curve api returns "Curve.fi DAI/USDC/USDT" while metaregistry returns "3pool".
     TODO: think of how to make pool names uniform."""
     with GlobalDBHandler().conn.read_ctx() as cursor:
-        existing_pools = [
-            string_to_evm_address(address)
-            for address in globaldb_get_general_cache_like(
-                cursor=cursor,
-                key_parts=(cache_type, str(inquirer.chain_id.serialize_for_db())),
+        existing_pools = {  # query the pools that we already have in the db
+            string_to_evm_address(address[0])
+            for address in cursor.execute(
+                'SELECT value FROM unique_cache WHERE key LIKE ?',
```

Comment:
why this change?

Reply:
We need to access the unique_cache instead of the general cache